src/style.css

```diff
@@ -1 +1,90 @@
-/* add styles here */
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir.ttc);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+.html {
```

Comment:
The selector `.html` seems to be incorrect if the intention is to style the HTML element. It should be `html` without the dot, as `.html` targets a class named 'html'.